app/serializers/site_serializer.rb

```diff
@@ -246,6 +247,10 @@ def include_anonymous_default_sidebar_tags?
     scope.anonymous? && !SiteSetting.legacy_navigation_menu? && SiteSetting.tagging_enabled && SiteSetting.default_sidebar_tags.present?
   end
 
+  def whispers_allowed_groups_names
+    SiteSetting.whispers_allowed_groups_map&.map { |id| Group.where(id: id).pluck_first(:name)  }
+  end
```

Comment:
I initially approved this @jancernik, but on second thought, we need to be a bit more careful here. As it is, this will expose the group names that are allowed to whisper to all users (including anonymous users). That would be a minor security issue, we would be disclosing to everybody the names of groups that may be private. 

Can we add a `include_whispers_allowed_groups_names?` method here that checks if current user can whisper? That would mean that the `whispers_allowed_groups_names` for the appropriate users.

Reply:
Good catch! There was a method already so I used that one instead. I also updated the tests.